<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Yatzy game</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dice_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dice_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dice_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dice_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dice_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="num_players_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>N. players:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_roll">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>New roll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_player">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Next player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lock_button_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lock_button_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lock_button_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lock_button_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lock_button_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock</string>
    </property>
   </widget>
   <widget class="QLabel" name="player_in_turn_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Player in turn:</string>
    </property>
   </widget>
   <widget class="QLabel" name="player_in_turn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rolls_done">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rolls done:</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_rolls_done">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rolls_left">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rolls left:</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_rolls_left">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="winner">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="winner_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Winner:</string>
    </property>
   </widget>
   <widget class="QLabel" name="seconds_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>sec</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_number_sec">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="dividing_line">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>16</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_number_min">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="minutes_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_of_players">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
